src/random.h

```diff
@@ -89,7 +89,7 @@ constexpr auto GetRandMillis = GetRandomDuration<std::chrono::milliseconds>;
  * is memoryless and should be used for repeated network events (e.g. sending a
  * certain type of message) to minimize leaking information to observers.
  *
- * The probability of an event occuring before time x is 1 - e^-(x/a) where a
+ * The probability of an event occurring before time x is 1 - e^-(x/a) where a
```

Comment:
Why only fix this one? The linter shows a few more.

```
bitcoin (master)₿ test/lint//lint-spelling.sh 
src/crypto/chacha_poly_aead.cpp:76: covert ==> convert
src/random.h:92: occuring ==> occurring
src/rpc/blockchain.cpp:794: nWe ==> new
src/util/syscall_sandbox.cpp:128: creat ==> create
test/functional/data/rpc_decodescript.json:81: ba ==> by, be
test/functional/data/rpc_decodescript.json:84: ba ==> by, be
^ Warning: codespell identified likely spelling errors. Any false positives? Add them to the list of ignored words in test/lint/lint-spelling.ignore-words.txt
```

Reply:
Most of that are false positives. Let me check.

Reply:
`covert` => `convert` is legit. Just checked it, will push here.

Reply:
Yes. When a pull fixes the spelling linter results, it usually ensures the result is clean in one commit and doesn't leave false positives.

Reply:
`nWe` => `new` is false positive because `nWe` is `\nWe must have...`

Reply:
Make sense, great. Just updated it, thanks!

Reply:
You didn't push fyi if you intended to

Reply:
I did it. Only covert and occuring are errors, other ones are false positives.

Reply:
Why not fix the false positives so the linter output is clean?